src/calculator.js

```diff
@@ -1,48 +1,45 @@
-exports._check = () => {
-  // DRY up the codebase with this function
-  // First, move the duplicate error checking code here
-  // Then, invoke this function inside each of the others
-  // HINT: you can invoke this function with exports._check()
-};
+exports._check = (x, y) => {
+
+    if (typeof x !== 'number') {
+      throw new TypeError(`${x} is not a number`);
+    }
+
+    if (typeof y !== 'number') {
+      throw new TypeError(`${y} is not a number`);
+    }
+
+}
+
 
 exports.add = (x, y) => {
-  if (typeof x !== 'number') {
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
+
+exports._check(x, y);
+
   return x + y;
+
 };
 
 exports.subtract = (x, y) => {
-  if (typeof x !== 'number') {
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
+
+  exports._check(x, y);
+
   return x - y;
+
 };
 
 exports.multiply = (x, y) => {
-  if (typeof x !== 'number') {
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
+  exports._check(x, y);
+
   return x * y;
+
 };
 
 exports.divide = (x, y) => {
-  if (typeof x !== 'number') {
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
+  
```

Comment:
Trailing spaces not allowed.

###### See http://eslint.org/docs/rules/no-trailing-spaces for details about this rule